Slim/Control/Commands.pm

```diff
@@ -3271,9 +3272,10 @@ sub _playlistXtracksCommand_parseSearchTerms {
 	my $sqlHelperClass = Slim::Utils::OSDetect->getOS()->sqlHelperClass();
 	
 	my $collate = $sqlHelperClass->collate();
-	
-	my $albumSort 
-		= $sqlHelperClass->append0("album.titlesort") . " $collate"
+
+	my $sortAlbumsByYear = $sortParam eq 'artflow' || $sortParam eq 'yearalbum' || $sortParam eq 'yearartistalbum';
```

Comment:
Please make this a regular `if () {} else {}` statement for easier readability.

Reply:
I assume this is mean for the "my $albumSort = condition ? a : b" ??? Never quite understood why so many people dislike the ternary operator... But still, I'll change this.

Reply:
I totally like the ternary operator - but not when it stretches over several lines. In this particular case there's a lots of concatenation done, too, which makes it a bit overwhelming. To me, at least.